apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -521,80 +518,72 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
                     </Heading>
-                    <Divider hidden/>
-                    { !applicationConfig.attributeSettings.advancedAttributeSettings.showMandateLinkedLocalAccount ?
-                        (<Field.CheckboxLegacy
-                            ariaLabel="Validate linked local account"
-                            name="validateLinkedLocalAccount"
-                            label={
-                                t("console:develop.features.applications.forms.advancedAttributeSettings." +
-                                "sections.linkedAccounts.fields.validateLocalAccount.label")
-                            }
-                            required={ false }
-                            value={
-                                initialSubject?.useMappedLocalSubject
-                                    ? [ "useMappedLocalSubject" ]
-                                    : []
-                            }
-                            readOnly={ readOnly }
-                            data-testid={ `${ testId }-validate-linked-local-account-checkbox` }
-                            hidden={
-                                !applicationConfig.attributeSettings.advancedAttributeSettings
-                                    .showValidateLinkedLocalAccount
-                            }
-                            hint={ t("console:develop.features.applications.forms.advancedAttributeSettings." +
-                            "sections.linkedAccounts.fields.validateLocalAccount.hint") }
-                        />):
-                        (<Field.CheckboxLegacy
-                            ariaLabel="Validate linked local account"
-                            name="validateLinkedLocalAccount"
-                            label={
-                                t("console:develop.features.applications.forms.advancedAttributeSettings." +
-                                "sections.linkedAccounts.fields.validateLocalAccount.label")
-                            }
-                            required={ false }
-                            value={
-                                initialSubject?.useMappedLocalSubject
-                                    ? [ "useMappedLocalSubject" ]
-                                    : []
-                            }
-                            readOnly={ readOnly }
-                            data-testid={ `${ testId }-validate-linked-local-account-checkbox` }
-                            hidden={
-                                !applicationConfig.attributeSettings.advancedAttributeSettings
-                                    .showValidateLinkedLocalAccount
-                            }
-                        />)
-                    }
                     <Field.CheckboxLegacy
-                        ariaLabel="Mandate linked local account"
-                        name="mandateLinkedLocalAccount"
+                        listen={ validateLinkedAccountCheckboxHandler }
```

Comment:
Can't we get this checkbox value through the `values` object inside the `submitValues` method and get rid of this custom listener?

Reply:
I need to conditionally disable each checkbox based on the value of the other one (when top one is unchecked bottom one should be disabled and when bottom one is checked top one should be disabled). For this i need to maintain the state of each checkbox